conf/nutch-default.xml

```diff
@@ -1874,6 +1874,72 @@ visit https://wiki.apache.org/nutch/SimilarityScoringFilter-->
   </description>
 </property>
 
+
+<!-- lib-htmlunit plugin properties; applies to protocol-htmlunit -->
+
+<property>
+  <name>htmlunit.page.load.delay</name>
```

Comment:
Among all, only 3 properties are shared between selenium and htmlunit. They are:

1. page.load.delay
2. take.screenshot
3. screenshot.location

Changes are updated in PR.